contracts/test/umbra-test.js

```diff
@@ -186,7 +186,7 @@ describe('Umbra', () => {
   it('should not let the eth receiver withdraw tokens', async () => {
     await expectRevert(
       ctx.umbra.withdrawToken(acceptor, ctx.token.address, { from: receiver1 }),
-      'Umbra: No tokens available for withdrawal'
+      'Umbra: No balance to withdraw or fee exceeds balance'
```

Comment:
It seems this error string was not changed in the code, so confused how these tests were passing before?

Reply:
When I added a shared internal withdraw message, I could no longer have a more specific error message for the non-meta withdrawal method, so the expected error for this test case changed, even though the behavior didn't.